packages/plugins/src/SidebarPlugin.ts

```diff
@@ -235,6 +235,14 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         return pagesByPriority;
       }
 
+      function moveRootPageDown(pagesByPriority) {
```

Comment:
We are modifying the data to fix a styling issue, which is not an ideal situation.
Someone would see it and might think it's a bug as no docs here, as to why we are doing it.

I wonder, if it's possible to do this without adding a root page as a child?

If we kept this code as it was. 
Do we just need to remove the margin from the first element in the vertical navigation?
We can then just modify the Vertical Navigation and keep the integrity of the data, so other sidebars don't have to treat element 0 as a special case.

As this is currently pro-sidebar.. suspect this is possible ?

Reply:
> We are modifying the data to fix a styling issue, which is not an ideal situation. Someone would see it and might think it's a bug as no docs here, as to why we are doing it.
> 
> I wonder, if it's possible to do this without adding a root page as a child?
> 
> If we kept this code as it was. Do we just need to remove the margin from the first element in the vertical navigation? We can then just modify the Vertical Navigation and keep the integrity of the data, so other sidebars don't have to treat element 0 as a special case.
> 
> As this is currently pro-sidebar.. suspect this is possible ?

I don't agree that it is a styling issue. It is not just that the first element has a different margin, it is because it shouldn't be the parent of the other pages.

In a folder like:

author/
|─ index.mdx/
|─ layouts.mdx
|─ modes-of-operation/
     |─ active-mode.mdx
     |─ index.mdx

we want the sidebar tree to look like this:

author
layouts
modes of operation
     active mode

currently it is returning:

author
  layouts
  modes of operation
       active mode

I think the sidebar plugin should return the correct json, rather than configuring the vertical navigation to treat the first item differently.

Reply:
interesting..what if we had a global sidebar created by using the glob '*' for root directories ?
That is what the F product wants I think ?

Reply:
> interesting..what if we had a global sidebar created by using the glob '*' for root directories ? That is what the F product wants I think ?

it would previously have been (using mosaic site as an example)

mosaic
─ quick start guides
─ ─ publish to aws
─ configure
─ ─ sources
─ ─ ─ local source
─ ─ ─ github repo
......

with these changes, every other page except mosaic is one level less indented so it would be like:

mosaic
quick start guides
─ publish to aws
configure
─ sources
─ ─ local source
─ ─ github repo
......